### PR TITLE
Add fields to Manifest form

### PR DESCRIPTION
**Pull request by jessieay**
- Going for parity with Sinatra app
- Still need to add ability to add multiple transporters / manifest items
- Also need to parse params correctly in controller, but this is close

### CODE REVIEW

**Comment by pkarman** [comment]
looks great. waiting for :green_heart: and will merge.

**Comment by jessieay** [comment]
@pkarman :green_heart: now -- darn shard failures!

File: app/controllers/manifests_controller.rb [hunk 1]
```diff
@@ -8,12 +8,10 @@ def create
     @manifest = Manifest.new(content: manifest_params)
 
     if @manifest.save && validate_manifest(manifest_params)
```
Comment (pkarman) [inline app/controllers/manifests_controller.rb:10]: this introduces a race condition. i.e., the manifest could be saved w/o being validated against the schema.

Look at https://github.com/18F/e-manifest/commit/aa7d449e2d456bf6c4ca0224f5ba17196d0701ec

Comment (jessieay) [inline app/controllers/manifests_controller.rb:10]: ok will update

Comment (pkarman) [inline app/controllers/manifests_controller.rb:10]: funny that we were both solving for this, in a similar way. :)

**Pull request merged**